src/ui/src/builder/journal/BuilderJournalEntry.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+	<div class="BuilderJournalEntry">
+		<div class="BuilderJournalEntry__summary">
+			<BuilderJournalEntryResult
+				:result="journalEntry.result"
+			></BuilderJournalEntryResult>
+			<div class="BuilderJournalEntry__summary__component">
+				{{ journalEntry.title }}
+			</div>
+		</div>
+		<div class="BuilderJournalEntry__info">
+			<div class="BuilderJournalEntry__info__datetime">
+				<div>{{ formattedDate }}</div>
+				<div class="BuilderJournalEntry__info__datetime__time">
+					{{ formattedTime }}
+				</div>
+			</div>
+			<div class="BuilderJournalEntry__info__details">
+				{{ journalEntry.trigger.type }}
+			</div>
+		</div>
+	</div>
+</template>
+
+<script setup lang="ts">
+import BuilderJournalEntryResult from "./BuilderJournalEntryResult.vue";
+import { JournalEntry } from "../BuilderJournal.vue";
+
+const props = defineProps<{
+	journalEntry: JournalEntry;
+}>();
+
+const dateObj = new Date(props.journalEntry.timestamp);
+
+const optionsDate: Intl.DateTimeFormatOptions = {
+	year: "numeric",
+	month: "short",
+	day: "numeric",
+};
+const formattedDate = dateObj.toLocaleDateString(undefined, optionsDate);
+
+const optionsTime: Intl.DateTimeFormatOptions = {
+	hour: "numeric",
+	minute: "2-digit",
+	second: "2-digit",
+	hour12: true,
+};
+const formattedTime = dateObj.toLocaleTimeString(undefined, optionsTime);
```

Comment:
You're losing reactivity here. It needs to be  computed.
```suggestion
const dateObj = computed(() => new Date(props.journalEntry.timestamp))

const optionsDate: Intl.DateTimeFormatOptions = {
	year: "numeric",
	month: "short",
	day: "numeric",
};
const formattedDate = computed(() => dateObj.value.toLocaleDateString(undefined, optionsDate));

const optionsTime: Intl.DateTimeFormatOptions = {
	hour: "numeric",
	minute: "2-digit",
	second: "2-digit",
	hour12: true,
};
const formattedTime = computed(() => dateObj.value.toLocaleTimeString(undefined, optionsTime));
```

It could be even better if you introduce a composable function like [`usePercentageFormatter`](https://github.com/writer/writer-framework/blob/5c15d13b14ccabc71d3cd0baf16d6d01fd3c7625/src/ui/src/composables/useFormatter.ts#L3-L3)